package.json

```diff
@@ -2,14 +2,14 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "jshint src && jscs src && pulp build --censor-lib --strict",
+    "build": "eslint src && pulp build -- --censor-lib --strict",
     "test": "pulp test"
   },
   "devDependencies": {
-    "jscs": "^2.8.0",
-    "jshint": "^2.9.1",
+    "eslint": "^3.19.0",
     "pulp": "^10.0.4",
-    "purescript-psa": "^0.3.9",
-    "rimraf": "^2.5.0"
+    "purescript": "^0.11.1",
+    "purescript-psa": "^0.5.0",
```

Comment:
No, in fact I figured that might be the problem - `npm` scripts prefer the path of `.bin` in `node_modules` over the global path.